src/PostgREST/App.hs

```diff
@@ -280,10 +280,10 @@ augumentRequestWithJoin schema allRels request =
 
 buildReadRequest :: [Relation] -> ApiRequest -> Either Text ReadRequest
 buildReadRequest allRels apiRequest  =
-  augumentRequestWithJoin schema rels =<< first formatParserError (foldr addFilter <$> (addOrder <$> readRequest <*> ord) <*> flts)
+  augumentRequestWithJoin schema rels
+  =<< first formatParserError (foldr addFilter <$> (foldr addOrder <$> readRequest <*> ords) <*> flts)
```

Comment:
and if you look at `=<<` as nothing more then monadic function composition, then you see why the indentation works as is :) (at least that's how i look at it)